testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/Suite6NonGeneric.java

```diff
@@ -47,7 +47,7 @@
                 testOnEnvironment(SinglenodeLdapAndFile.class).withGroups("ldap", "ldap_and_file", "ldap_cli", "ldap_and_file_cli").build(),
                 testOnEnvironment(SinglenodeLdapInsecure.class).withGroups("ldap").build(),
                 testOnEnvironment(SinglenodeLdapReferrals.class).withGroups("ldap").build(),
-                testOnEnvironment(SinglenodeLdapBindDn.class).withGroups("ldap").build(),
+                testOnEnvironment(SinglenodeLdapBindDn.class).withGroups("ldap").withExcludedGroups("ldap_multiple_binds").build(),
```

Comment:
The BindDn env uses a fixed bind user which cannot be used together with bind patterns obviously. So an exclusion was needed.